app/views/customers/new.html.slim

```diff
@@ -33,7 +31,13 @@
                     br
                     small Step 3 Tempoary Address
             .stepContainer
-              = form_for @customer, url: customer_index_path do |f|
+              = form_for @customer, url: customers_path  do |f|
```

Comment:
@RaviMariya-BTC If you are using restful routes then no need to have the URL argument for the form.